src/faq.html

```diff
@@ -103,6 +103,12 @@
           <p>
             The Duracell CR2032 shipped with Home Assistant Yellow is estimated to last 10 years or more (with self-discharge taken into account). Replacement likely won't be necessary within the product's lifetime. If your RTC does not show the correct date, this is likely because the initial time synchronization failed. Make sure that your Internet router does not block NTP (Network Time Protocol) traffic.
           </p>
+      - title: >
+            What is the purpose of the 3.5 mm barrel audio jack?
+        content: >
+          <p>
+            The 3.5 mm barrel audio jack can be connected to an amplifier and used in combination with the VLC add-on. This exposes the output as a media player in Home Assistant Core.
```

Comment:
The media player doesen't appear automatically, so maybe worth mentioning/linking the VLC add-on here: https://www.home-assistant.io/integrations/vlc_telnet#home-assistant-add-on